docs/conf.py

```diff
@@ -21,7 +21,7 @@
 project = 'METcalcpy'
 copyright = '2019, NCAR'
 author = 'UCAR/NCAR, NOAA, CSU/CIRA, and CU/CIRES'
-author_list = 'Win-Gildenmeister, M., Burek, T., Fillmore, D., Halley Gotway, J., Adriaansen, D., Fisher, H.'
+author_list = 'Win-Gildenmeister, M., Burek, T., Fillmore, D., Adriaansen, D., Fisher, H.'
```

Comment:
Dan Adriaansen needs to be removed from the author list, he has not contributed to the METcalcpy repository or docs.

Reply:
Done